src/Components/Color/ColorCard.jsx

```diff
@@ -47,7 +59,11 @@ export default function ColorCard({ color, onDelete, onUpdate }) {
           <p className="color-card-headline">{color.hex}</p>
           <p>{color.role}</p>
           <p>contrast: {color.contrastText}</p>
-
+          {color.contrastStatus && (
+            <p style={getContrastMessageStyle()}>
+              Overall Contrast Score: {color.contrastStatus}
+            </p>
+          )}
```

Comment:
All these elements don't change with the "isEditing" state. You can move them before the ternary operator and avoid the duplication, since you have the same html twice here.